torch/storage.py

```diff
@@ -87,6 +87,10 @@ def bool(self):
         """Casts this storage to bool type"""
         return self.type(type(self).__module__ + '.BoolStorage')
 
+    def quint8(self):
```

Comment:
if it's never used - delete? casting to qint types doesn't make sense as we'd need to provide quantizer

Reply:
OK will remove